extensions/binding/org.eclipse.smarthome.binding.iota/src/main/java/org/eclipse/smarthome/binding/iota/handler/TransformationServiceProvider.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.iota.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.transform.TransformationService;
+
+/**
+ * Provide a transformation service which can be used during Mqtt topic transformation.
+ *
+ * @author Simon Kaufmann - initial contribution and API
```

Comment:
author

Reply:
please review the profit from this interface. I guess (as in the MQTT binding) you use it only at one specific place where in the end a static method on the `TransformationHelper` is called. Just refactor this to simply use the `TransformationHelper`

Reply:
First part done.
As for the second remark: it is used in the HandlerFactory, and it uses a `bundleContext` reference. I'm not sure if I can replace the call by a static one wherever needed in the code because of this. Wdyt ?